ARCs/arc-0028.md

```diff
@@ -0,0 +1,81 @@
+---
+arc: 28
+title: Algorand Event Log Spec
+status: Draft
+discussions-to: https://github.com/algorandfoundation/ARCs/issues/113
+---
```

Comment:
Please fill this part as explained in the new [ARC-0](https://github.com/algorandfoundation/ARCs/blob/main/ARCs/arc-0000.md)

---
arc: <to be assigned>
title: <The ARC title is a few words, not a complete sentence>
description: <Description is one full (short) sentence>
author: <a comma-separated list of the author's or authors' name + GitHub username (in parenthesis), or name and email (in angle brackets).  Example, FirstName LastName (@GitHubUsername), FirstName LastName <foo@bar.com>, FirstName (@GitHubUsername) and GitHubUsername (@GitHubUsername)>
discussions-to: <URL>
status: Draft
type: <Standards Track, Meta, or Informational>
category (*only required for Standards Track): <Interface, or ARC>
created: <date created on, in ISO 8601 (yyyy-mm-dd) format>
requires (*optional): <ARC number(s)>
---

Reply:
👍 Updated